app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -172,6 +172,10 @@ struct DeviceDelegateOculusVR::State {
     vrapi_SetPropertyInt(&java, VRAPI_BLOCK_REMOTE_BUTTONS_WHEN_NOT_EMULATING_HMT, 0);
     // Reorient the headset after controller recenter.
     vrapi_SetPropertyInt(&java, VRAPI_REORIENT_HMD_ON_CONTROLLER_RECENTER, 1);
+    // Enable fixed-foveated rendering.
+    // TODO - This should be user configurable. VRAPI_SYS_PROP_FOVEATION_AVAILABLE
+    //        can be checked to determine if the option should be presented.
+    vrapi_SetPropertyInt(&java, VRAPI_FOVEATION_LEVEL, 2);
```

Comment:
I think it's better to enable this only for the chrome scene. You can do that via`DeviceDelegateOculusVR::SetRenderMode()`

According to the spec foveated rendering for WebVR should be up to the JS code right? via WebGL extensions or WebXR when available.

Reply:
Agree, we should probably have this preference affect the browser's 2d modes separately.

There is no defined API for setting foveated level in the WebXR spec yet.  Unlike MRS (Mixed resolution shading) and LMS (Lens Matched Shading), this kind of fixed-foveated-rendering does not need changes to the content to function.  Turning it on manually for immersive content could make the difference between being able to see some immersive sites and it not working.  Perhaps we should give the user choice in this case.